src/IW/Effects/Download.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE AllowAmbiguousTypes  #-}
+
+module IW.Effects.Download where
+
+import Network.HTTP.Client (Manager, Request (..), Response (..), httpLbs)
+import Network.HTTP.Types (Status (..))
+
+import IW.App (App (..), grab, Has, WithError)   
```

Comment:
Is `App (..)` necessary here? Wouldn't just `App` do?

Reply:
@vrom911 Yep, App is just fine.